### ISSUE TITLE
prompt_embeds_attention_mask shape issue

### ISSUE THREAD

**Issue by quantumiracle** [PKU-YuanGroup/Open-Sora-Plan#317]
Hi, 

In the sampling code, if prompt_embed is given instead of the prompt, why is the attention mask same shape as the prompt_embeds? https://github.com/PKU-YuanGroup/Open-Sora-Plan/blob/b08681f697658c81361e1ec6c07fba55c79bb4bd/opensora/sample/pipeline_videogen.py#L209

For example, prompt_embed can have shape (1,1,L,D), while the attention_mask is supposed to be of shape (1, L).
